reporter/urlparams.js

```diff
@@ -85,7 +85,7 @@ function getUrlParams() {
 			// Allow just a key to turn on a flag, e.g., test.html?noglobals
 			value = param.length === 1 ||
 				decodeQueryParam( param.slice( 1 ).join( "=" ) ) ;
-			if ( urlParams[ name ] ) {
+			if ( name in urlParams ) {
```

Comment:
We can also use `Object#hasOwnProperty` here, I'm not a big fan of the in operator and we don't need to search the property in the urlParams prototype, as it is null.

If you prefer to keep the in operator, I'm ok with it, there's nothing wrong with it.